notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,1541 @@
+       "      <td>15:2638:0</td>\n",
+       "      <td>16:2639:0</td>\n",
+       "      <td>17:2640:0</td>\n",
+       "      <td>18:2641:0</td>\n",
+       "      <td>19:2642:0</td>\n",
+       "      <td>20:2643:0</td>\n",
+       "      <td>21:2644:0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>2</th>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1:3:1</td>\n",
+       "      <td>2:944:1</td>\n",
+       "      <td>3:2626:0</td>\n",
+       "      <td>4:2627:0</td>\n",
+       "      <td>5:2628:0</td>\n",
+       "      <td>6:2629:0</td>\n",
+       "      <td>7:2630:1</td>\n",
+       "      <td>8:2631:0</td>\n",
+       "      <td>9:2632:0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>12:2635:0</td>\n",
+       "      <td>13:2636:0</td>\n",
+       "      <td>14:2637:0</td>\n",
+       "      <td>15:2638:0</td>\n",
+       "      <td>16:2639:0</td>\n",
+       "      <td>17:2640:0</td>\n",
+       "      <td>18:2641:0</td>\n",
+       "      <td>19:2642:0</td>\n",
+       "      <td>20:2643:0</td>\n",
+       "      <td>21:2644:0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>3</th>\n",
+       "      <td>3.0</td>\n",
+       "      <td>1:4:1</td>\n",
+       "      <td>2:944:1</td>\n",
+       "      <td>3:2626:0</td>\n",
+       "      <td>4:2627:0</td>\n",
+       "      <td>5:2628:0</td>\n",
+       "      <td>6:2629:0</td>\n",
+       "      <td>7:2630:1</td>\n",
+       "      <td>8:2631:0</td>\n",
+       "      <td>9:2632:0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>12:2635:0</td>\n",
+       "      <td>13:2636:0</td>\n",
+       "      <td>14:2637:0</td>\n",
+       "      <td>15:2638:0</td>\n",
+       "      <td>16:2639:0</td>\n",
+       "      <td>17:2640:0</td>\n",
+       "      <td>18:2641:0</td>\n",
+       "      <td>19:2642:0</td>\n",
+       "      <td>20:2643:0</td>\n",
+       "      <td>21:2644:0</td>\n",
+       "    </tr>\n",
+       "    <tr>\n",
+       "      <th>4</th>\n",
+       "      <td>5.0</td>\n",
+       "      <td>1:5:1</td>\n",
+       "      <td>2:944:1</td>\n",
+       "      <td>3:2626:0</td>\n",
+       "      <td>4:2627:0</td>\n",
+       "      <td>5:2628:0</td>\n",
+       "      <td>6:2629:0</td>\n",
+       "      <td>7:2630:1</td>\n",
+       "      <td>8:2631:0</td>\n",
+       "      <td>9:2632:0</td>\n",
+       "      <td>...</td>\n",
+       "      <td>12:2635:0</td>\n",
+       "      <td>13:2636:0</td>\n",
+       "      <td>14:2637:0</td>\n",
+       "      <td>15:2638:0</td>\n",
+       "      <td>16:2639:0</td>\n",
+       "      <td>17:2640:0</td>\n",
+       "      <td>18:2641:0</td>\n",
+       "      <td>19:2642:0</td>\n",
+       "      <td>20:2643:0</td>\n",
+       "      <td>21:2644:0</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "<p>5 rows × 22 columns</p>\n",
+       "</div>"
+      ],
+      "text/plain": [
```

Comment:
@yexing99 in the settings here, user, item and genres are treated as fields, and the features in each field should be counted in the dictionary. Not sure if it is right to restart the count from 1 from item field to genre field. 

Add @Leavingseason